aquality-selenium-template-cucumber/src/test/java/aquality/selenium/template/cucumber/features/Demo.feature

```diff
@@ -12,4 +12,10 @@ Feature: Demo
       | Comment  | I'd like to contact you! |
     And I accept Privacy and Cookies Policy
     And I click Send button
-    Then Notification about empty fields is present
\ No newline at end of file
+    Then Notification about empty fields is present
+
+  @demo
+  Scenario: ScenarioContext demo
+    Given I store '2' as 'value1'
+    And I store '3' as 'value2'
```

Comment:
"I store '2' as 'value1'" looks like a When step. The proper time of "Given" keyword is present perfect or the verb in passive voice, smth like `Given The value '2' is stored as 'value1'` or `I stored '2' as 'value1'`
Also we should add tabulation before `And` step.
And regarding `Then` step, it should define expected state, e.g. `Then The result of adding 'value1' to 'value2' is '5'`

Reply:
It is just an example